### PR TITLE
update for both iso6 and 7

### PR DESCRIPTION
**Pull request by wouterdb**

### CODE REVIEW

File: Networking/Cisco/project.yml [hunk 1]
```diff
@@ -6,7 +6,11 @@ license: ASL 2.0
 copyright: 2021 Inmanta
 modulepath: libs
 downloadpath: libs
-repo:
-    - https://github.com/inmanta/
 install_mode: release
-requires:
+# The repo block is for ISO6
```
Comment (sanderr) [inline Networking/Cisco/project.yml:10]: I would also mention the inmanta or inmanta-core version for each of these that is not specifically lsm.

File: Networking/Cisco/project.yml [hunk 2]
```diff
@@ -6,7 +6,11 @@ license: ASL 2.0
...
+repo:
+  - type: package
+    url: https://packages.inmanta.com/<token>/inmanta-service-orchestrator-6-stable/python/simple/
```
Comment (sanderr) [inline Networking/Cisco/project.yml:13]: If I understand correctly this used to be OSS, only using PyPi and public GitHub. Now it requires a token. Can we use the quickstart repo instead?

File: Networking/Cisco/project.yml [hunk 3]
```diff
@@ -6,7 +6,11 @@ license: ASL 2.0
...
+# The pip block is for ISO7 and up
+pip:
+  index_url: https://packages.inmanta.com/<token>/inmanta-service-orchestrator-6-stable/python/simple/
```
Comment (sanderr) [inline Networking/Cisco/project.yml:16]: iso6 repo for iso7 and up. Probably no longer relevant depending on how the other thread resolves.

File: Networking/Cisco/project.yml (hunk 2 above)
Comment (edvgui) [inline Networking/Cisco/project.yml:13]: Does the `<token>` do something?  Is this project supposed to be usable "as is"?

File: Training/project/srlab/project.yml [hunk 4]
```diff
@@ -6,12 +6,17 @@ license: ASL 2.0
...
       url: https://pypi.org/simple/
     - type: git
       url: https://github.com/inmanta/examples
-install_mode: release
-requires:
+
+# The pip block is for ISO7 and up
+pip:
+  index_url: https://packages.inmanta.com/public/quickstart/python/simple/
+  extra_index_url: ["https://pypi.org/simple/"]
```
Comment (sanderr) [inline Training/project/srlab/project.yml:22]: Do we need PyPi here? I see it's also present in the package sources but I think it's not required, and insecure. Iirc the quickstart repo inherits from PyPi.

Same question for the others.

**Review by sanderr — CHANGES_REQUESTED** [review]

File: Networking/Cisco/project.yml (hunk 2 above)
Comment (wouterdb) [reply Networking/Cisco/project.yml:13]: this requires a user to add an access token, ...

lsm is not publicly available.

Should I add a comment to clarify this?

Comment (wouterdb) [reply Networking/Cisco/project.yml:13]: this always requires non-public modules

Comment (wouterdb) [reply Networking/Cisco/project.yml:13]: Should I add something like

```
# This example requires licensed modules, 
# replace <token> with inmanta access token you received with your license
```

File: Networking/Cisco/project.yml (hunk 1 above)
Comment (wouterdb) [reply Networking/Cisco/project.yml]: added, I hope correctly

File: Networking/Cisco/project.yml (hunk 3 above)
Comment (wouterdb) [reply Networking/Cisco/project.yml]: updated

File: Training/project/srlab/project.yml (hunk 4 above)
Comment (wouterdb) [reply Training/project/srlab/project.yml]: removed

**Review by sanderr — APPROVED** [review]

File: Networking/Cisco/project.yml (hunk 2 above)
Comment (edvgui) [reply Networking/Cisco/project.yml:13]: Yes, that sounds like a good enough solution

**Review by edvgui — CHANGES_REQUESTED** [review]
cf latest comment

**Review by edvgui — APPROVED** [review]

**Pull request merged**